nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -434,11 +442,15 @@ bool RegulatedPurePursuitController::isCollisionImminent(
   return false;
 }
 
-bool RegulatedPurePursuitController::inCollision(const double & x, const double & y)
+bool RegulatedPurePursuitController::inCollision(
+  const double & x,
+  const double & y,
+  const double & theta)
 {
-  unsigned int mx, my;
+// Assumes setFootprint already set
+  unsigned int wx, wy;
```

Comment:
Revert to `mx` and `my` the `m` means "map" and the `w` means world. Since this is converting world coordines (x,y) into map coordinates (mx,my) they should use the `m` version

Reply:
I know its not used, but in case its used later, we don't want it to be unclear and cause a bug